README.md

```diff
@@ -84,6 +84,14 @@ You can setup automatic purging by adding the `GuardianDb.ExpiredSweeper` as a w
   worker(GuardianDb.ExpiredSweeper, [])
 ```
 
+If you are working with a production release using Distillery, you will need to take an additional step and add `guardian_db` and `distillery` to your `applications` list in order to have the sweeper available:
```

Comment:
To avoid the double ands how about:

"If you are working with a production release using Distillery, you need ensure both `guardian_db` and `distillery` are added to your `applications` list."